diff --git a/wazo_lib_rest_client/client.py b/wazo_lib_rest_client/client.py
index c684c96..a616594 100644
--- a/wazo_lib_rest_client/client.py
+++ b/wazo_lib_rest_client/client.py
@@ -3,6 +3,8 @@
 # SPDX-License-Identifier: GPL-3.0+
 
 import logging
+import os
+import sys
 
 from functools import partial
 from requests import HTTPError
@@ -78,7 +80,9 @@ class BaseClient(object):
 
     def session(self):
         session = Session()
-        session.headers = {'Connection': 'close'}
+
+        user_agent = os.path.basename(sys.argv[0])
+        session.headers = {'Connection': 'close', 'User-agent': user_agent}
 
         if self.timeout is not None:
             session.request = partial(session.request, timeout=self.timeout)